config/routes.rb

```diff
@@ -81,6 +81,10 @@
     get "/:slug", to: "step_nav#show"
   end
 
+  get "/world/:name/news(.:locale).atom",
```

Comment:
Probably more for my understanding than anything since I'm not super familiar with all the routes stuff - why do we have to define this atom route explicitly for world location news but not for topical events? Organisations also does things differently to both but I'm assuming that's because it supports non English feeds

Reply:
I tried the same approach as topical events, but the `.atom` got captured as `:locale`.

I can't quite understand what's happening with organisations, as it seems rather more complicated than it needs to be.

Reply:
Ah, I see, that makes sense re topical events vs these